lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-google-cloud_manager-virtual_template.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Google_CloudManager_VirtualTemplate < MiqAeServiceManageIQ_Providers_CloudManager_Template
```

Comment:
Can you all the # rubocop:disable Metrics/LineLength to clear out that Rubocop flag?